### PR TITLE
Add SpeechBubble

### PR DESCRIPTION
**Pull request by Roohn**
# Description
Add SpeechBubble with the possibility to set the tail direction.

## Screenshots
image
<hr/>

image

### CODE REVIEW

**Review by RobVermeer — APPROVED** [review]
😍

File: src/components/SpeechBubble/index.js [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+    `};
+`
+
+const SpeechBubbleContent = styled.div`
+  position: relative;
+  display: inline-block;
+  padding: ${space[16]};
+  color: ${color.space};
+  font-size: ${fontSize[16]};
+  line-height: ${lineHeight[1.25]};
```
Comment (rppld) [inline src/components/SpeechBubble/index.js]: https://github.com/TicketSwap/solar/blob/master/src/theme.js#L160-L163
```suggestion
  line-height: ${lineHeight.title};
```

**Review by rppld — APPROVED** [review]
One small comment 👌

**Pull request merged**